### ISSUE TITLE
Make New Version

### ISSUE THREAD

**Issue by sysads** [MediaMonks/tilt-game-android#5]
Hi

Am i allowed to recreate this game with a different twist using a different game engine and publish?

**Comment by stephan-mediamonks** [comment]
Yes you're allowed to do this, as per the license attached to the source code. 

Please let us know when it's released, we'd love to see (and play) it!
